docs/add-webhook-to-slack.md

```diff
@@ -2,16 +2,16 @@
 
```

Comment:
Maybe start with an introductory paragraph (just a sentence or two) outlining what we are doing (creating a Slack Incoming WebHook) and why it is necessary (to allow our OpenWhisk action to post messages to Slack)